### DISCUSSION TITLE
Only things that's keeping me from switching to Next player

### DISCUSSION THREAD

**Discussion by anoshione** [anilbeesetti/nextplayer#766] [Ideas]
- Online subtitle. Honestly its a pain to to pause a movie download a subtitle, extract it, then add it. It would be really convenient to have a online subtitle option if possible.
- Option to select and delete bulk items. 

Is it possible to implement these features, please?

**Comment by anilbeesetti** [comment]
sure @anoshione. It's in the todo list but will take some time as i'm only free on weekends to work on the app
